<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paymentDialog</class>
 <widget class="QDialog" name="paymentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>paymentDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border-image: url(:/masterDesktop/Resources/complaints_bg.png);
}
#btn_close{
border-image: url(:/masterDesktop/Resources/close.png);
}
#btn_close:pressed{
margin:1px;
}
#wgt_title{
background:#002E29;
border:1px solid #00947E;
}
#lbl_title{
font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#FFFFFF;
}</string>
     </property>
     <widget class="QPushButton" name="btn_close">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>28</y>
        <width>37</width>
        <height>37</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>37</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>86</y>
        <width>412</width>
        <height>424</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wgt_title" native="true">
         <property name="minimumSize">
          <size>
           <width>410</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>410</width>
           <height>42</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lbl_title">
            <property name="text">
             <string>在线支付</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>380</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>380</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>21</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:14px;
color:#FFFFFF;</string>
              </property>
              <property name="text">
               <string>您一共需支付：</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_sum">
              <property name="styleSheet">
               <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:18px;
color:#2EB4AF;
padding-left:20px;</string>
              </property>
              <property name="text">
               <string>200元</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_content">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#ACACAC;</string>
            </property>
            <property name="text">
             <string>充值成功后可得120元网费</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>6</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="qrCodeGenerateWgt" name="wgt_code" native="true">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>130</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>130</width>
                <height>130</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>9</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#ACACAC;</string>
            </property>
            <property name="text">
             <string>支持支付宝和微信扫码</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>74</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qrCodeGenerateWgt</class>
   <extends>QWidget</extends>
   <header>qrCodeGenerateWgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="masterDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
